saw-script/src/SAWScript/REPL/Command.hs

```diff
@@ -6,8 +6,6 @@ Maintainer  : huffman
 Stability   : provisional
 -}
 {-# LANGUAGE OverloadedStrings #-}
--- TODO RGS: Do better (or at least comment why we do this)
```

Comment:
Thank you for fixing this! I'm embarrassed that I accidentally checked this comment in :)